libminifi/test/bustache-tests/CMakeLists.txt

```diff
@@ -33,7 +33,6 @@ FOREACH(testfile ${BUSTACHE_INTEGRATION_TESTS})
   createTests("${testfilename}")
   target_link_libraries(${testfilename} ${CATCH_MAIN_LIB})
   target_link_libraries(${testfilename} ${Boost_IOSTREAMS_LIBRARY})
-  target_link_libraries(${testfilename} bustache)
```

Comment:
I don't understand this change. Why do we no longer need this library?

Reply:
We never needed this actually. We add minifi-bustache-extensions library as a whole archive which links the locally built BUSTACHE::libbustache bundled library. This line would additionally try to link the bustache library installed on the system.